### PR TITLE
Update SendGrid cohort export docs

### PR DESCRIPTION
**Pull request by ampcynthli**
# Amplitude Developer Docs PR

## Description

- reword some points and slight refactor
- add new consideration

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/destinations/sendgrid-cohort.md [hunk 1]
```diff
@@ -14,17 +14,20 @@ This SendGrid integration allows you to send audiences from Amplitude to SendGri
...
-- The recommended way to guarantee that all email addresses are unique is by setting your Amplitude User IDs to use email addresses. 
-- If you chose User ID, make sure that all User ID fields contain valid emails or the sync can't complete.
+- Cohorts cannot have duplicate emails, as it will cause sync errors with SendGrid.
+  - The recommended way to guarantee unique email addresses is by setting your Amplitude User IDs to use email addresses. 
+- All emails that are being exported must be valid; otherwise, the sync will fail.
 - You must map the email, first name, and last name fields. If the Amplitude user doesn't have values for the first and last name fields, the corresponding SendGrid contact won't have first or last names, only the email address.
 - In cases where users don't have an email address, they aren't synced to SendGrid. This can cause a discrepancy between the number of users in SendGrid and Amplitude. 
+- Emails on SendGrid's side is **not** case-sensitive. Both User@Company.com and user@company.com will sync to the same contact (user@company.com)
```
Comment (caseyamp) [inline docs/data/destinations/sendgrid-cohort.md:22]:
```suggestion
- In SendGrid, emails aren't case-sensitive. For example, "User@Company.com" and "user@company.com" sync to the same contact (user@company.com).
```

File: docs/data/destinations/sendgrid-cohort.md [hunk 2]
```diff
@@ -14,17 +14,20 @@ This SendGrid integration allows you to send audiences from Amplitude to SendGri
...
+- New contacts will be created for Amplitude users that are not SendGrid contacts, existing contacts will not be duplicated. Both types of users will be added to the list.
```
Comment (caseyamp) [inline docs/data/destinations/sendgrid-cohort.md:23]:
```suggestion
- New contacts are created for Amplitude users that are not SendGrid contacts, and existing contacts aren't duplicated. Both types of users are added to the list.
```

**Review by caseyamp — COMMENTED** [review]
@ampcynthli this is great -- I made some minor suggestions to improve accessibility and align more with our style guidelines. Thank you for the work! 

I'm explicitly not selecting 'Request changes' because of a bug that breaks the AWS preview build -- when you've committed my suggestions, re-request review and I'll approve. Sorry for the trouble.

**Comment by ampcynthli** [comment]
Thanks so much for your help, Casey.

**Review by caseyamp — APPROVED** [review]

**Comment by caseyamp** [comment]
@ampcynthli I think the build bug popped up anyway. LMK when you want me to merge it and I can force it.

**Pull request merged**